lib/rubygems/sigstore/rekor/log_entry.rb

```diff
@@ -0,0 +1,32 @@
+class Gem::Sigstore::Rekor::LogEntry
+
+  def self.from(entry_response)
```

Comment:
`class << self` is recommended at shopify and we use that style in this codebase as well.

Reply:
It seems like a code smell that the from method is longer than the rest of the class. I think the class would be a bit easier to understand if you would separate this class into an entry and logentry struct.

Reply:
I chose `def self.the_method` since that is what rubygems does.

![image](https://user-images.githubusercontent.com/33674553/143040131-4a0a11e8-7345-4965-98ab-b7fe19192fea.png)

Reply:
> I chose def self.the_method since that is what rubygems does.

Yeah, I'm not a big fan of that and I would like it to change. The main reason is that `def self.foo` is always public even if a private specifier is used, which isn't ideal. I think this is good reason to diverge from the rubygems standard code style.